### ISSUE TITLE
Feature: "handpower" any object requied mechanical energy

### ISSUE THREAD

**Issue by elfl0rd** [StrangeLoopGames/EcoSuggestions#871]
It's really necessary, especially in the early stages.
Please add ability to "power" one item by "attaching" player.
Move close to workbench, e.g. water pump or mill, etc.
Press button like "power" or "work"
Now player power this workbench like mechanical power source.
Calories spend per time, if no calories, or player stop\left - power stops.

(Dont know about best place for this, maybe suggestions better...)

**Comment by johnkslg** [comment]
Yes! Been planning this. Thinking you charge it up by clicking it with calories, and it runs for a long while.

**Comment by elfl0rd** [comment]
I think it's a better option with an "attached" player after all.
At the same time it will allow to use "free" players in the big community :P

**Comment by newzealdan** [comment]
Yes! I was thinking the same about workbenches. I thought about how it would be great if you could birth NPCs to power work benches too. The NPCs need homes and food to work. This drives more demand for resources in the economy too.

**Comment by newzealdan** [comment]
And having higher tier housing and food means the NPC can work on higher tier benches

**Comment by johnkslg** [comment]
NPCs would be a huge addition, moves away from the 'everyone is a real person' aspect of the game, so it would be a big change to consider.

**Comment by newzealdan** [comment]
I bet! I'm sure you can work out the technicalities. In the worlds I've played there haven't ever been enough players to create a dominating demand on the environment, and certainly in real life its a combination of the number of people all demanding a high standard of living that causes the environmental strain.
